### PR TITLE
Required yet-to-be-defined terms

### PR DESCRIPTION
**Pull request by mih**
This list is incomplete, but we definitey need these terms defined already.

Ping #21

### CODE REVIEW

**Comment by mih** [comment]
@jsheunis yes, please!

File: datalad_tabby/resources/contexts/common.jsonld [hunk 1]
```diff
@@ -10,5 +10,23 @@
...
   "license": "schema:license",
   "name": "schema:name",
+  "url": "schema:url",
   "version": "schema:version",
+  "address": "",
+  "affiliation": "",
+  "annex-key": "",
+  "author": "",
+  "citation": "",
+  "checksum[md5]": "",
```
Comment (jsheunis) [inline datalad_tabby/resources/contexts/common.jsonld:20]: I want to include the term from https://ontobee.org/ontology/NCIT?iri=http://purl.obolibrary.org/obo/NCIT_C171276

Here's more info about the ontology: https://ontobee.org/ontology/NCIT

However, I am unsure how to add the the key-value pair for the specific vocabulary in this context. Would it be something like: `"obo": "http://purl.obolibrary.org/obo/"`, and then the term is `"checksum[md5]": "obo:NCIT_C171276"`?

File: datalad_tabby/resources/contexts/common.jsonld [hunk 2]
```diff
@@ -10,5 +10,23 @@
...
+  "date": "",
+  "email": "",
+  "funder": "",
+  "funding": "",
+  "orcid": "",
+  "path[POSIX]": "",
+  "project[code]": "",
+  "publication": "",
+  "sample[organism]": "",
+  "size[bytes]": "",
```
Comment (jsheunis) [inline datalad_tabby/resources/contexts/common.jsonld:31]:
```suggestion
  "affiliation": "schema:affiliation",
  "annex-key": "",
  "author": "schema:author",
  "citation": "schema:citation",
  "checksum[md5]": "",
  "contact": "schema:contactPoint",
  "date": "",
  "email": "schema:email",
  "funder": "schema:funder",
  "funding": "schema:funding",
  "orcid": "",
  "path[POSIX]": "",
  "project[code]": "",
  "publication": "schema:publication",
  "sample[organism]": "",
  "size[bytes]": "",
```

File: datalad_tabby/resources/contexts/common.jsonld [hunk 3]
```diff
@@ -10,5 +10,23 @@
...
+  "url": "schema:url",
   "version": "schema:version",
+  "address": "",
+  "affiliation": "",
+  "annex-key": "",
+  "author": "",
+  "citation": "",
+  "checksum[md5]": "",
+  "contact": "",
+  "date": "",
```
Comment (jsheunis) [inline datalad_tabby/resources/contexts/common.jsonld:22]: Does this refer to a general `date` property or to the datatype (e.g. https://schema.org/Date)

Comment (mih) [reply datalad_tabby/resources/contexts/common.jsonld:22]: The question alone is an indication that this name is not good. It is presently used for publications. It better becomes

https://schema.org/datePublished

File: datalad_tabby/resources/contexts/common.jsonld (hunk 1 above)
Comment (mih) [reply datalad_tabby/resources/contexts/common.jsonld:20]: yes, looks good

File: datalad_tabby/resources/contexts/common.jsonld (hunk 3 above)
Comment (jsheunis) [reply datalad_tabby/resources/contexts/common.jsonld:22]: we could also consider a naming schema with the square brackets similar to `project[code]` or `sample[organism]`, where the part in brackets defines a specific type/flavor/category of date, e.g.: 
```
"date[published]": "schema:datePublished",
"date[start]": "schema:startDate"
```
or these could just be semantically separate keys.

**Comment by mih** [comment]
I am putting this back in draft mode. The whole semantics issues is not thought through enough and needs more design.

**Comment by mih** [comment]
With #86, we no longer need a global context. Most of this PR will move into a standard set of sheet classes instead.

**Pull request closed without merging**